packages/xstate-svelte/jest.config.js

```diff
@@ -0,0 +1,15 @@
+const { jest: lernaAliases } = require('lerna-alias');
+
+module.exports = {
+  moduleNameMapper: lernaAliases(),
+  transform: {
+    '^.+\\.svelte$': [
+      'svelte-jester',
+      {
+        preprocess: true,
+        rootMode: 'upward'
```

Comment:
isn't this an option related to Babel? we are not quite using it on the master branch, is it needed?

Reply:
`rootMode: 'upward'` is to allow a `svelte.config.js` inside a workspace folder rather than just at the project root. See https://github.com/mihar-22/svelte-jester/pull/15 and https://github.com/mihar-22/svelte-jester/issues/12

Reply:
TIL, quite weird - it kinda works differently than a very similar option in Babel 🤷‍♂️

Reply:
Yes, I agree the naming is confusing.